ietf/templates/submit/submission_status.html

```diff
@@ -130,6 +132,11 @@ <h2 class="mt-5">Submission checks</h2>
         <p class="alert alert-warning my-3">
             This submission is awaiting the first draft upload.
         </p>
+    {% elif submission.state_id == 'validating' %}
+        <p class="alert alert-warning my-3">
+            This submission is still being processed and validated. This normally takes a few minutes after
+            submission. Please contact the secretariat if it has been more than an hour.
```

Comment:
On this, I've added an indication of how long it's been since submission. That can be done with a small template change and I think it's enough of a hint.